docker/run_web.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Runs the webapp in gunicorn.
+
+PORT=${PORT:-8000}
+echo "Running webserver on http://localhost:${PORT} ..."
+
+gunicorn \
+	--pythonpath /app \
+	--error-logfile=- \
+	--access-logfile=- \
+	--log-file=- \
+	--config=python:ichnaea.webapp.settings \
+	--bind 0.0.0.0:"${PORT}" \
```

Comment:
In Socorro-land, we have configuration for a few other things:

```
GUNICORN_WORKERS=${GUNICORN_WORKERS:-"1"}
GUNICORN_WORKER_CLASS=${GUNICORN_WORKER_CLASS:-"sync"}
GUNICORN_MAX_REQUESTS=${GUNICORN_MAX_REQUESTS:-"10000"}
GUNICORN_MAX_REQUESTS_JITTER=${GUNICORN_MAX_REQUESTS_JITTER:-"1000"}
```

In Ichnaea, that's in `ichnaea.webapp.settings`. I think we should move it here so it can be set by environment variables and managed along with the infra code. I'm going to push those changes off to a new PR.